lib/GLPI/Agent/SNMP/MibSupport/BrotherNetConfig.pm

```diff
@@ -77,6 +81,34 @@ sub getModel {
     return $model;
 }
 
+sub getPortIfType {
+    my ($self) = @_;
+
+    my $device = $self->device
+        or return;
+
+    # Get list of device ports
+    my $ports = $device->{PORTS}->{PORT};
+
+    # Get list of device ports types (lan(1)/wirelesslan(2))
+    my $brMultiIFType = $self->walk(brMultiIFType) || {};
+
+    # Get list of device ports names
+    my $brMultiIFNodeType = $self->walk(brMultiIFNodeType) || {};
```

Comment:
Here it would be simpler to just return as the result will be the same: nothing would be changed.
```suggestion
    # Get list of device ports names
    my $brMultiIFNodeType = $self->walk(brMultiIFNodeType)
        or return;
```